contents/blog/brand.md

```diff
@@ -0,0 +1,152 @@
+
+It was going to be free on launch day, but we knew users would still be wary of wasting their time deploying a piece of software produced by a two-person startup. 
+
+Being open source helped. People could inspect the code for themselves, but we concluded the majority would turn away _at_ the website, the docs, and the readme before glancing at a line of code.
+
+That's why we set out to create a website and brand that felt both unique and intentional, including a [public handbook](/handbook) that explained what we were doing, why, and how.
+
+As a counter example, an obviously templated or one-pager style website signals:
+
+1. **The product may not be secure.** If a startup doesn't care about having their own website design, will they care about security? A lack of demonstrated competence shows a lack of attention to detail.
+
+2. **The team behind the product isn't very strong.** The product probably won't improve quickly, or become a "must use" tool. Those startup launches that everyone hears about? They've put energy into their appearance.
+
+3. **The product may not exist in a few more weeks.** Why should a potential customer invest time in a product if it's obvious the team making it threw its website together in a day, or ripped it from an existing template?
+
+So, as frustrating as it might be to spend extra time and effort on something "superficial" like a website, remember you're providing your product to human beings. Everyone judges books by their covers.
+
+## Your company needs to stand for something
+
+Take two examples:
+
+* Open-source product analytics
+* We equip every developer to build successful products
+
+There is a trade-off between the two.
+
+The first is very clear. We started with it when this is what we had.
+
+As we became more well-known, and as we added lots of products into our platform, we shifted to the latter. This is a better reflection of _why_ our company exists – it is more than just a piece of software. 
+
+We introduced it because the company was getting larger, and the scope of what we were setting out to achieve had expanded. We now have 8+ products, and we produce a ton of content to help engineers build better products. 
+
+The need for total clarity wasn't as important later on, relative to the need for us standing for something. It was necessary to start small then widen.
+
+## You need to create your brand for someone...
+
+One of the most important things we did was to [create an Ideal Customer Profile](/newsletter/ideal-customer-profile-framework).
+
+When we launched, we weren't sure quite how focused we'd need to be on engineers vs product managers. We realized, after launching, that we could go all in on engineers and technical founders, who make up around 80% of our users. Our brand and product has become more oriented towards these users as a result.
+
+Here are some specific ways we got more specific for this audience after we clarified it:
```

Comment:
clarified feels weird here. zeroed-in on it? focused on it?

Reply:
and is the rep on 'specific' intentional? feels a bit clumsy

Reply:
it is not